src/components/button/button.tsx

```diff
@@ -165,11 +165,12 @@ export const Button = forwardRef<HTMLButtonElement | HTMLAnchorElement, IButtonP
         className,
         children,
         state,
+        disabled,
         ...otherProps
     } = props;
 
     const isOnlyIcon = children == null || children === '';
-    const isDisabled = state === 'disabled' || state === 'loading';
+    const isDisabled = disabled === true || state === 'loading';
```

Comment:
This will need to be changed as well once you've converted `state` to a designated property